src/eventthread.cpp

```diff
@@ -105,6 +105,8 @@ EventThread::EventThread()
       showCursor(false)
 {}
 
+SDL_DisplayMode dm = {0};
```

Comment:
What is this for?

Reply:
It 's an initial sdl query.
https://wiki.libsdl.org/SDL_DisplayMode
notifies the thread of the size and position (in screen coordinates) of the window.

Reply:
Yes, you query it. Where is the code that uses the size and position?

Reply:
Apparently in the previous PR: #240. I guess it has been mistakenly added to this one.

Reply:
Rechecked, and you are right.
Those 3 queries are not needed for this script (there is no code about size and position in it).